redirects/api/keras_nlp/layers/fnet_encoder/index.html

```diff
@@ -0,0 +1,2 @@
+
+<meta http-equiv="refresh" content="0; URL='https://keras.io/api/keras_nlp/modeling_layers/'" />
```

Comment:
Is there a way to test this out?

Reply:
this should be `/modeling_layers/fnet_encoder`?

No idea how to test - -"